scss/civicrm/common/_base.scss

```diff
@@ -263,3 +263,45 @@
     color: $gray-darker;
   }
 }
+
+.page-civicrm:not(.page-civicrm-contact-view) {
```

Comment:
so i can guess that this is the generic style for pages with a certain structure, but it's basically impossible to understand that from just looking at the file. I think this needs either a separate partial, or to be placed in a already existing partial that at least is pertinent with what the style is about. Either way, it definitely needs some comment with details

Reply:
Ok I'ill do it accordingly with proper comments